coc.html

```diff
@@ -168,11 +168,11 @@ <h3><b>License</b></h3>
 
           <ul class="list-inline">
 
-            <li><img src="img/fb.png" alt="image"></li>
+            <li><a href="https://www.facebook.com/PyConIndia" target="_blank"><img src="img/fb.png" alt="image"></a></li>
 
-            <li><img src="img/twitt.png" alt="image"></li>
+            <li><a href="https://twitter.com/pyconindia" target="_blank"><img src="img/twitt.png" alt="image"></a></li>
 
-          <ul>
```

Comment:
Can you please add mailing list button also as in index.html page?

Reply:
done